tests/README.md

```diff
@@ -1 +1,21 @@
-Until we have actual tests, this is a placeholder so Git keeps the directory.
+# Phyloreferencing Test Suite
+
+## Usage
+> py.test tests/
+
+## Description
+This test suite is designed to be used to validate the outputs of the `phylo2owl.py` script. It does this in two ways:
+
+ 1. By comparing an output file with its expected output, and reporting an error if they differ.
+ 2. By validating an output OWL file against a set of SHACL shapes.
+
+You can examine [previous runs of this test suite](https://travis-ci.org/phyloref/phylo2owl) on Travis-CI.
+
+## Test scripts
+
+ - `conftest.py`: A py.test configuration file that sets up the fixtures.
```

Comment:
That *would* be a better name, but unfortunately `conftest.py` is the hardcoded configuration file name for py.test. On the plus side, anybody familiar with py.test should know what it's for, and it's documented here and [in the file itself](https://github.com/phyloref/phylo2owl/pull/19/files#diff-484462fced51d1a06b1d93b4a44dd535). We could *also* create a `fixtures.py` file containing the fixtures as a py.test module, but [we'd need a `conftest.py` file](https://github.com/pytest-dev/pytest/issues/1569#issuecomment-221790555) to load that plugin, which so we can't really get around it.